src/fsharp/vs/Symbols.fs

```diff
@@ -523,10 +523,22 @@ and FSharpEntity(cenv:cenv, entity:EntityRef) =
     member x.RecordFields = x.FSharpFields
     member x.FSharpFields =
         if isUnresolved() then makeReadOnlyCollection[] else
+    
+        if entity.IsILEnumTycon then
+            let (TILObjectReprData(scoref,enc,tdef)) = entity.ILTyconInfo
+            let formalTypars = entity.Typars(range.Zero)
+            let formalTypeInst = generalizeTypars formalTypars
+            let ty = TType_app(entity,formalTypeInst)
+            let formalTypeInfo = ILTypeInfo.FromType cenv.g ty
+            tdef.Fields.AsList
```

Comment:
I think this can be done using GetILFieldInfosOfType on an infoReader, e.g. https://github.com/fsharp/FSharp.Compiler.Service/blob/0892b3c45ee4d3467a89e56b483919bb0271f5c2/src/fsharp/NicePrint.fs#L1491

(It's not too bad as is, but the uniform way is preferable)